playbook.yml

```diff
@@ -111,8 +111,8 @@
       group: root
       mode: '0755'
     loop:
-      - /etc/containers
-      - /etc/containers/systemd
+    - /etc/containers
```

Comment:
I think this change was already merged in a separate PR

Reply:
Indeed, I already rebased.